.github/workflows/markdownlint.yml

```
@@ -0,0 +1,21 @@
+name: Markdown Lint
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)